main.go

```diff
@@ -151,10 +156,16 @@ func main() {
 	leaseDuration := time.Second * 137
 	renewDeadline := time.Second * 107
 	retryPeriod := time.Second * 26
+
+	optionsTlSOptsFuncs := []func(*tls.Config){
+		func(config *tls.Config) { minTlsDefault(config) },
+	}
+
 	mgrOptions := ctrl.Options{
 		Scheme:                 scheme,
 		MetricsBindAddress:     metricsAddr,
 		Port:                   webhookPort,
+		TLSOpts:                optionsTlSOptsFuncs,
```

Comment:
Does this configure the TLS version on the webhook server as well? 


```
mgrOptions := ctrl.Options{
		WebhookServer:
```

Reply:
yes, i tested it on my local, and i check from sslscan. Later i attached printscr here